src/auth/src/credentials/util/jws.rs

```diff
@@ -29,32 +28,28 @@ pub(crate) const CLOCK_SKEW_FUDGE: Duration = Duration::from_secs(10);
 pub(crate) const DEFAULT_TOKEN_TIMEOUT: Duration = Duration::from_secs(3600);
 
 /// JSON Web Signature for a token.
-#[derive(Clone, Serialize, Default, Builder)]
-#[builder(setter(into, strip_option), default)]
-pub struct JwsClaims<'a> {
-    pub iss: &'a str,
+#[derive(Serialize)]
+pub struct JwsClaims {
+    pub iss: String,
```

Comment:
I like the simplicity of using `String`.  It is more expensive, but this function will be called like once per hour so we should not complicate things too much in search of small performance improvements.

Reply:
Ack